### ISSUE TITLE
[Swift] Introduce swiftlint for generated code

### ISSUE THREAD

**Issue by d-date** [swagger-api/swagger-codegen#6993]
##### Description
The code generated by mastache is too much warned by swiftlint. We should run `swiftlint autocorrect` after generated codes.

##### Swagger-codegen version

2.3.0

##### Suggest a fix/enhancement

When language is swift3/4, we should run `swiftlint autocorrect`.

**Comment by wing328** [comment]
@d-date thanks for the suggestio, which sounds good to me to leverage some third-party tools to improve the code format.

I might be able to come up with something this coming weekend. Will keep you posted.

cc @jgavris @ehyche @Edubits @jaz-ah

**Comment by jgavris** [comment]
It of course depends on your swiftlint configuration. I would encourage you to ignore in swiftlint any code that isn't your own. We do that for example for our generated swagger clients.

**Comment by eoinnorris** [comment]
I agree with jgarvis. Swiftlint is fine but it is dogmatic, on where colons should be placed, commas etc. I tend to abide by it for my code. but there is not need to fix up the generated code.
